tests/functional/test_install_check.py

```diff
@@ -1,8 +1,10 @@
 from tests.lib import create_test_package_with_setup
 
 
-def contains_expected_lines(string, expected_lines):
-    return set(expected_lines) <= set(string.splitlines())
+def assert_contains_expected_lines(string, expected_lines):
+    lines = string.splitlines()
+    for expected_line in expected_lines:
+        assert any(line.endswith(expected_line) for line in lines)
```

Comment:
This won't give us a very useful failure message.

I suggest adding a f"{line!r} not found in:\n{string}" to this assert.

Reply:
Oh no you merged this before I was able to response :smile:  Would you like a follow up on this?

Reply:
Go ahead!

Reply:
I've just followed this up in GH-8692.